peps/pep-0770.rst

```diff
@@ -659,6 +530,17 @@ Conditional project source SBOM files
 How can a project specify an SBOM file that is conditional? Under what
 circumstances would an SBOM document be conditional?
 
+Statically defining source SBOM files
+-------------------------------------
+
+How can SBOM files within a project source (and outside a project source tree)
+be specified statically such that files are added to ``.dist-info/sboms``?
+
+A previous version of this PEP suggested a mechanic for
+adding files into subdirectories under ``.dist-info/`` using a top-level
+``pyproject.toml`` table ``[dist-info.files]``. This addition has been
+refactored out of this PEP and will be proposed in a separate PEP.
```

Comment:
If this won't make it into this PEP then it isn't an open issue (i.e. something that needs to be settled before the PEP is considered ready for consideration), but a "Deferred Idea".